client/bio-webclient/src/main/kotlin/ac/uk/ebi/biostd/client/integration/web/SubmitClient.kt

```diff
@@ -86,6 +88,17 @@ interface GeneralOperations {
     fun addUserInGroup(groupName: String, userName: String)
 }
 
+interface StatsOperations {
+    fun getStatsByAccNo(accNo: String): List<SubmissionStat>
+    fun getStatsByType(type: String): List<SubmissionStat>
+    fun getStatsByTypeAndAccNo(type: String, accNo: String): SubmissionStat
+
+    fun registerStat(stat: SubmissionStat): Unit
+    fun registerStats(type: String, statsFile: File): List<SubmissionStat>
+    fun incrementStats(type: String, statsFile: File): List<SubmissionStat>
+    fun increment(file: File): Unit
```

Comment:
In a previous PR I added an operation to the stats resource in order to [calculate the submission files size](https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/stats/web/StatsResource.kt#L71)

Would you mind adding this operation to the client as well? I think it would be super useful to have it as a CLI operation

Reply:
create [ticket](https://www.pivotaltracker.com/story/show/185880630) so we also add an IT

Reply:
I've renamed the ticket to CLI operation so it's clearer what's covering